spring-cloud-starter-task-composedtaskrunner/src/main/java/org/springframework/cloud/task/app/composedtaskrunner/DataFlowConfiguration.java

```diff
@@ -84,4 +125,31 @@ private void validateUsernamePassword(String userName, String password) {
 			throw new IllegalArgumentException("A username may be specified only together with a password");
 		}
 	}
+
+	@Configuration
+	@Conditional(OnOAuth2ClientCredentialsEnabled.class)
+	static class clientCredentialsConfiguration {
+		{
+			System.out.println("Client Credentials Enabled");
```

Comment:
Need a logger here instead of System.out.println

Reply:
Removed.